src/main/java/org/isf/serviceprinting/gui/MedicalStockSelection.java

```diff
@@ -501,5 +509,21 @@ public void actionPerformed(ActionEvent e) {
 	public void actionPerformed(ActionEvent e) {
 		formatSelected = e.getActionCommand();
 	}
+	
+	public static void main(String[] args) {
```

Comment:
this main can be removed too.
(we can figure out another quick way of testing GUI windows - using main methods is not the best solution)